rasa/core/policies/memoization.py

```diff
@@ -239,20 +256,56 @@ def predict_action_probabilities(
         return self._prediction(result)
 
     def _metadata(self) -> Dict[Text, Any]:
-        return {
-            "priority": self.priority,
-            "max_history": self.max_history,
-            "lookup": self.lookup,
-        }
+        return {"config": self.config, "lookup": self.lookup}
```

Comment:
can we please drop the `config` here as we store it with the graph schema? The only thing we actually need to store is the lookup table

Reply:
In the super class it looks like the config is loaded from the metadata file too

Reply:
[here](https://github.com/RasaHQ/rasa/blob/fdb0413301dcd3021ae3742d0d91528dc94a9f8d/rasa/core/policies/policy.py#L444) ?

Reply:
Very good point. How about nuking the entire `load` implementation of the parent class and make it an abstract method so that subclasses have to implement it themselves?